### PR TITLE
fixes #183 - removes speculate & express dependencies

### LINKED ISSUE #183: Vulnerability (Time of Check Time of Use)
**Is your feature request related to a problem? Please describe.**

The `speculate` module has a dependency on `chownr`, which has a Medium Severity vulnerability.

screen shot 2018-12-11 at 11 45 36

Details: https://snyk.io/vuln/npm:chownr:20180731

Have raised: https://github.com/bbc/speculate/issues/60

### PR DESCRIPTION
**Pull request by ChrisBAshton**
Resolves #183 

_We no longer deploy to Cosmos, so no need for Speculate, and no need for server.js (therefore no need for express)_

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,9 +11,7 @@
     "postinstall": "npm run install:packages",
     "prepublish": "npm run build",
     "publish": "lerna publish",
-    "start": "node server.js",
     "storybook": "npm run clear_styled_components && NODE_ENV=development start-storybook -p 8080 -c .storybook",
-    "storybook:export": "build-storybook -c .storybook -o public",
     "speculate": "speculate",
```
Comment (dr3) [inline package.json]: 😱

Comment (ChrisBAshton) [reply package.json]: This isn't needed! 😄 We call `run-storybook` which has its own call to `build-storybook`.

`storybook:export` is another artefact left over from the Speculate days.

Comment (dr3) [reply package.json]: Oh did I click the wrong line, I meant the `"speculate": "speculate"` under it

Comment (ChrisBAshton) [reply package.json]: oh! yeah that shouldn't be there 😂

**Review by dr3 — DISMISSED** [review]

File: packages/components/example-button/package-lock.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
 		},
 		"core-js": {
 			"version": "1.2.7",
-			"resolved": "https://registry.npmjs.org/core-js/-/core-js-1.2.7.tgz",
+			"resolved": "http://registry.npmjs.org/core-js/-/core-js-1.2.7.tgz",
```
Comment (bcmn) [inline packages/components/example-button/package-lock.json]: It doesn't feel like this file should have changed?

Comment (ChrisBAshton) [reply packages/components/example-button/package-lock.json]: Don't get me started 😂 

Hopefully after https://github.com/BBC-News/psammead/pull/190 is merged I'll update this PR and everything will be fine.

**Review by bcmn — APPROVED** [review]

**Review by sareh — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**